### PR TITLE
Manual add-in of @rbayersdorfer-newrelic's edits

### PR DESCRIPTION
**Pull request by dbarnesbrownNR**
Little background: This one languished for a bit because the release wasn't quite ready to go, but this is a busy page in terms of frequent edits so we had a pretty gnarly merge conflict. Could have been resolved using the less-hacky method, buuut the visual editing task was super confusing, so I have manually added the changes reflected in [https://github.com/newrelic/docs-website/pull/2286/files] to the current version of the page, and will be placing this in draft mode to get a quick validation of changes from @rbayersdorfer-newrelic.

### CODE REVIEW

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx [hunk 1]
```diff
@@ -227,11 +227,10 @@ Every NRQL query will begin with a `SELECT` statement or a `FROM` clause. All ot
         id="extrapolate-example-1"
         title="Example of extrapolating throughput"
       >
-        A query that will show the extrapolated throughput of a service named `interestingApplication`.
+        A query that will show the extrapolated throughput of a service named 'interestingApplication'.
 
         ```
-        SELECT count(*) FROM Transaction WHERE appName='interestingApplication' 
-        SINCE 60 minutes ago EXTRAPOLATE
+        SELECT count(*) FROM Transaction WHERE appName='interestingApplication'
```
Comment (rbayersdorfer-newrelic) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx:233]: This does not seem related to the original PR

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx [hunk 2]
```diff
@@ -790,24 +789,18 @@ Every NRQL query will begin with a `SELECT` statement or a `FROM` clause. All ot
...
 
-            `appName RLIKE 'z.*|q.*''`
+            `appName RLIKE r'z.*|q.*''`
 
             * `z-app`
             * `q-app`
-
-            `hostname RLIKE 'ip-10-351-[0-2]?[0-9]-.*'`
-
+            *`hostname RLIKE r'ip-10-351-[0-2]?[0-9]-.*'`
```
Comment (rbayersdorfer-newrelic) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx:796]: I don't think we want a `*` here for the bullet point, this should be above the bulleted section

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx [hunk 3]
```diff
@@ -1736,6 +1728,70 @@ SELECT histogram(duration, 10, 20) FROM PageView SINCE 1 week ago
...
     </CollapserGroup>
   </Collapser>
+
+  <Collapser
+      className="freq-link"
+      id="func-capture"
+      title={<InlineCode>capture(attribute, regular expression)</InlineCode>}
+    >
+      Use the `capture()` to extract values from an attribute using a regular expression. Uses [RE2 syntax](https://github.com/google/re2/wiki/Syntax).
+      It takes two arguments:
```
Comment (rbayersdorfer-newrelic) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx:1738]: Do we want a space/extra line break between lines 1737 and 1738?

**Review by rbayersdorfer-newrelic — COMMENTED** [review]
Looks good, just a few comments 👍

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx (hunk 2 above)
Comment (dbarnesbrownNR) [reply src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx:796]: Got it, will make the change!

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx (hunk 1 above)
Comment (dbarnesbrownNR) [reply src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx:233]: You're right this was a hiccup from combing through lines of text. Fixing!

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx [hunk 4]
```diff
@@ -790,24 +789,19 @@ Every NRQL query will begin with a `SELECT` statement or a `FROM` clause. All ot
...
 
-            `appName RLIKE 'z.*|q.*''`
+            `appName RLIKE r'z.*|q.*''`
+            
+            `hostname RLIKE r'ip-10-351-[0-2]?[0-9]-.*'`
 
             * `z-app`
             * `q-app`
-
-            `hostname RLIKE 'ip-10-351-[0-2]?[0-9]-.*'`
```
Comment (rbayersdorfer-newrelic) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx:798]: I think we still want this line here,  not above. It relates to the bullets below it. We just don't want it as one of the bullet points.

Comment (dbarnesbrownNR) [reply src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx:798]: Fixed!

**Review by paperclypse — APPROVED** [review]
Looks good to me!
